tests/integration/project_test.py

```diff
@@ -67,7 +87,30 @@ def test_volumes_from_container(self):
             client=self.client,
         )
         db = project.get_service('db')
-        self.assertEqual(db.volumes_from, [data_container])
+        self.assertEqual(db._get_volumes_from(), [data_container.id])
+
+        project.kill()
+        project.remove_stopped()
```

Comment:
These two lines wont run if the test fails. The test case should take care of this for you, so you can remove it.